src/components/GlobalSignIn/blocks/Login.tsx

```diff
@@ -0,0 +1,49 @@
+import { faSignIn } from "@fortawesome/free-solid-svg-icons";
+import { P1, Button, AnchorButton } from "@deskpro/deskpro-ui";
+import { nbsp } from "../../../constants";
+import { Invalid } from "../../common";
+import type { FC } from "react";
+import type { AnyIcon } from "@deskpro/deskpro-ui";
+import type { Maybe } from "../../../types";
```

Comment:
I strongly recommend that we switch to use absolute/alias paths like `@/types/` and maybe only exporting one thing per file, like `@/common/Invalid`.
```
import { faSignIn } from "@fortawesome/free-solid-svg-icons";
import { P1, Button, AnchorButton } from "@deskpro/deskpro-ui";
import { nbsp } from "@/constants"; // @not sure that this needs to be a const.
import Invalid from "@/common/Invalid";
import type { FC } from "react";
import type { AnyIcon } from "@deskpro/deskpro-ui";
import type Maybe from "@/types/Maybe";
```
The alias definitely makes the code more maintainable and subjectively I think the same is true for default exports.

Reply:
here story for aliases https://app.shortcut.com/deskpro/story/157041/lansweeper-update-to-use-aliases